### DISCUSSION TITLE
Feature Request: Sound when test run or deployment completes

### DISCUSSION THREAD

**Discussion by parkerbrown11** [forcedotcom/salesforcedx-vscode#5370] [Ideas]
**Is your feature request related to a problem? Please describe.**
When running apex tests and deploying source to org, it can take several seconds and sometimes minutes. I often get distracted multi-tasking and forget to come back to check on the status of the test or deployment until much later.

**Describe the solution you'd like**
_A clear and concise description of what you want to happen._
I would like to be able to turn on/off a sound when the testing or deployment completes.

**Describe alternatives you've considered**
_A clear and concise description of any alternative solutions or features you've considered._
None

**Additional context**
_Add any other context or screenshots about the feature request here._

**Comment by diyer** [comment]
@parkerbrown11 Thanks for the idea! We are going to convert open this up to a discussion to other users.

**Comment by AllanOricil** [comment]
This have to be a priority. I wanna play this [song](https://youtu.be/dQw4w9WgXcQ?feature=shared) when my tests are done.
